src/Raven.Server/Smuggler/Documents/DatabaseSmuggler.cs

```diff
@@ -83,7 +83,7 @@ private static void EnsureStepProcessed(SmugglerProgressBase.Counts counts)
 
         private void ProcessType(DatabaseItemType type, SmugglerResult result, BuildVersionType buildType)
         {
-            if ((_options.OperateOnTypes & type) != type)
+            if ((_options.OperateOnTypes & type) != type && type != DatabaseItemType.LegacyAttachments) //TODO:Add legacy attachment option in the studio
```

Comment:
We won't be adding this, I think. It would uglify us for everything.

Reply:
Ok